src/browser.ts

```diff
@@ -28,6 +28,4 @@ export * from './types';
 
 export * from './walkTree';
 
-// XXX remove in the next breaking semver change (3.0)
-const compose = require('lodash.flowright');
-export { compose };
+export { default as compose } from './utils/flowRight';
```

Comment:
```suggestion
export { compose } from './utils/flowRight';
```

Reply:
On second thought, how about just inlining the `flowRight.ts` module into this module?